Sources/TuistLoader/Models+ManifestMappers/Target+ManifestMapper.swift

```diff
@@ -219,7 +219,13 @@ extension XcodeGraph.Target {
             )
         } ?? [])
 
-        for sourceFile in allSources {
+        let scriptsSources = try manifest.scripts.flatMap(\.outputPaths)
```

Comment:
Is this a legit place to create references to script files?

Reply:
This makes an assumption that all script build phases produce outputs that need to be included as sources. I don't think that's a safe assumption to make - e.g. outputs could be resources or files that don't need to be included in the final target at all such as documentation.

I'd avoid implicitly trying to add sources in this manner, would look for ways to make it more explicit, such as allowing defining the sources for the generated files:

Can follow the same idea of not globbing variable paths

```swift
.target(
  // ...
  sources: [
      "$(SRCROOT)/Generated/SomeGeneratedFile.swift",
  ]
)
```

or something more explicit:

```swift
.target(
  // ...
  sources: [
      .generated("Generated/SomeGeneratedFile.swift"),
  ]
)
```